README.md

```diff
@@ -35,7 +35,7 @@ the [Spine Examples][spine-examples] GitHub organization.
 
 Building Spine modules and libraries requires JDK 8. 
 
-The support of JDK 11 is planned for the [version 3.0][v3] of the framework.
+The support of JDK 11 is planned for the [version 2.0][v2] of the framework.
```

Comment:
We actually work with Java 11 and Spine 1.7.x. By support, do we mean that the framework is going to be compiled with Java 11 source code target?